### PR TITLE
iOS 13+ support

### PR DESCRIPTION
**Pull request by kaler-earnin**
- Fix statusBarOrientation issue for iOS 13+
- Bump version to 5.4.2

### CODE REVIEW

**Comment by kaler-earnin** [comment]
@mal3k are you able to review this and accept this patch?  It is to support iOS 13 deprecation.  It's a fairly straightforward change that is backwards compatible.

**Comment by jgavris** [comment]
+1 We'd love to make this warning go away. I reviewed as well and it is implemented as we would expect.

**Review by DmitryPashinskiy — APPROVED** [review]

**Comment by ahujamanish** [comment]
Are there any updates on this? This is a blocker for us.

File: SupportSDK.framework/Headers/ZDKUIUtil.h [hunk 1]
```diff
@@ -221,8 +218,16 @@ CGCenterRectInRect(CGRect rect, CGRect inRect)
...
 ZDKUIIsLandscape()
 {
-    UIInterfaceOrientation orientation = [UIApplication sharedApplication].statusBarOrientation;
-    return UIInterfaceOrientationIsLandscape(orientation);
+    if #available(iOS 13.0, *) {
+                return UIApplication.shared.windows
+                    .first?
+                    .windowScene?
+                    .interfaceOrientation
+                    .isLandscape ?? false
```
Comment (jgavris) [inline SupportSDK.framework/Headers/ZDKUIUtil.h:226]: This is Swift in an Objective-C header...it will need a little touch-up.

**Comment by RonanMcH** [comment]
Hi @kaler-earnin , this is a public repository for a closed source binary. Any changes will need to be put into our private repository and can't be merged here. We have a ticket open for this issue so I will close this PR.
Thanks,
Ronan.

**Pull request closed without merging**